functional-tests/index.js

```diff
@@ -3,14 +3,15 @@
 const Browser = require('./lib/browser');
 const App = require('./lib/app');
 const assert = require('assert');
+const config = require('./config.js');
 
 describe('tests', () => {
   let browser;
   let app;
   let port;
 
   before(() => {
-    app = App(require('./apps/default')).listen();
+    app = App(require('./apps/default')).listen(config.port);
     port = app.address().port;
```

Comment:
You don't really need this port variable anymore since it's hardcoded, but it's not doing any real harm, just a little messier than it needs.